db/migrate/20180130165200_drop_webmks.rb

```diff
@@ -0,0 +1,5 @@
+class DropWebmks < ActiveRecord::Migration[5.0]
+  def up
+    SettingsChange.where(:resource_type => 'MiqServer', :key => '/server/remote_console_type', :value => 'MKS').delete_all
```

Comment:
The `SettingsChange` class needs to be defined here as well if you want to use it.